src/GitParamTabExpansion.ps1

```diff
@@ -76,7 +76,7 @@ $longGitParams = @{
     revert = 'edit mainline no-edit no-commit gpg-sign signoff strategy= strategy-option continue quit abort'
     rm = 'force dry-run cached ignore-unmatch quiet'
     shortlog = 'numbered summary email format='
-    show = 'pretty= format= abbrev-commit no-abbrev-commit oneline encoding= notes no-notes show-notes no-standard-notes standard-notes show-signature'
+    show = 'pretty= format= abbrev-commit no-abbrev-commit oneline encoding= expand-tabs no-expand-tabs notes no-notes show-notes no-standard-notes standard-notes show-signature name-only name-status stats shortstats numstats'
```

Comment:
Only `stat`!
```suggestion
    show = 'pretty= format= abbrev-commit no-abbrev-commit oneline encoding= expand-tabs no-expand-tabs notes no-notes show-notes no-standard-notes standard-notes show-signature name-only name-status stat shortstat numstat'
```

It occurs to me that the common [diff formatting](https://git-scm.com/docs/git-show#_diff_formatting) arguments could probably be centralized in a variable that can be reused for `diff`, `log`, `git stash show`, etc.